extensions/wikia/Sitemap/SpecialSitemap_body.php

```diff
@@ -246,7 +231,6 @@ private function generateNamespace( Array $sitemapIndex = null, $forceUpdate = f
 			$namespaceSitemap = $this->generateNamespaceFromDb();
 			if( !empty($namespaceSitemap) ) {
 				// this sitemap was pre-generated by the maintenance script and stored in db
-				wfProfileOut( __METHOD__ );
```

Comment:
removing wfProfileIn() line 228 and wfProfileOut() in line 306 in generateNamespace() function ?